src/second-peek.md

```diff
@@ -34,7 +34,7 @@ error[E0507]: cannot move out of borrowed content
 *Sigh*. What now, Rust?
 
 Map takes `self` by value, which would move the Option out of the thing it's in.
-Previously this was fine because we had just `take`n it out, but now we actually
+Previously this was fine because we had just taken it out, but now we actually
```

Comment:
I defer to the author but I think this may be correct. I believe they're referencing the method [`.take`](https://doc.rust-lang.org/beta/std/option/enum.Option.html#method.take) that was used and simply appending an "n" to informally make the tense correct.